### DISCUSSION TITLE
Moosefs4  how to open the EC

### DISCUSSION THREAD

**Discussion by Songdantes** [moosefs/moosefs#597] [Q&A]
<img width="1429" alt="image" src="https://github.com/user-attachments/assets/5f471105-6b44-4966-9a34-64d3f165f712">
<img width="1146" alt="image" src="https://github.com/user-attachments/assets/6f5a4e62-e603-41cc-9f21-e2cf71fd8a41">

I deployed 5 machines, but the CGI shows that EC is not open. How can I configure it to open EC.

**Comment by chogata** [comment]
The same issue was raised in discussion #584 I'm closing this one.
